common.yml

```
@@ -476,365 +476,365 @@ translations:
     - key: options.job_title.cto
       t: CTO
     - key: options.job_title.front_end_developer
-      t: Front End Developer/Engineer
+      t: Desenvolvedor ou Engenheiro Front End
     - key: options.job_title.back_end_developer
-      t: Back End Developer/Engineer
+      t: Desenvolvedor ou Engenheiro Back End
     - key: options.job_title.full_stack_developer
-      t: Full Stack Developer/Engineer
+      t: Desenvolvedor ou Engenheiro Full Stack
     - key: options.job_title.web_developer
-      t: Web Developer
+      t: Desenvolvedor Web
     - key: options.job_title.web_designer
       t: Web Designer
     - key: options.job_title.ux_designer
       t: UX Designer
     - key: options.job_title.ui_designer
       t: UI Designer
     - key: options.job_title.back_end_dev
-      t: Back-end Developer/Engineer
+      t: Desenvolvedor ou Engenheiro Back End
 
     # JavaScript proficiency
     - key: options.javascript_proficiency.0
-      t: Not able to write any JavaScript
+      t: Não sou capaz de escrever javascript
     - key: options.javascript_proficiency.1
-      t: Writing short, simple JavaScript or jQuery statements
+      t: Eu escrevo código simples em JavaScript ou jQuery
     - key: options.javascript_proficiency.2
-      t: Working on existing front-end codebases using modern frameworks (React, Vue, etc.)
+      t: Trabalho em projetos front-end usando framework modernos (React, Vue, etc.)
     - key: options.javascript_proficiency.3
-      t: Architecturing entire front-end codebases from scratch
+      t: Crio projetos front-end inteiros do zero
     - key: options.javascript_proficiency.4
-      t: Handling advanced front-end patterns (state management, data loading, etc.)
+      t: Lido com de padrões avançados de front-end (gerenciamento de estado, carregamento de dados, etc.)
 
     # CSS proficiency
     - key: options.css_proficiency.0
-      t: Virtually no knowledge of CSS
+      t: Não tenho nenhum conhecimento de CSS
     - key: options.css_proficiency.1
-      t: Using CSS frameworks and tweaking existing styles
+      t: Uso estruturas CSS e ajustando estilos existentes
```

Acredito que não há a necessidade de traduzir _framework_, nunca vi ninguém falando "estrutura" ou algo similar.

```suggestion
      t: Uso frameworks CSS e aprimoro estilos existentes
```